tests/integration/redis-cli.tcl

```diff
@@ -451,31 +451,33 @@ if {!$::tls} { ;# fake_redis_node doesn't support TLS
         assert_error "*tag not match*" {run_cli_with_input_pipe X "echo foo" set key wrong_tag}
     }
 
-    test "DUMP RESTORE with -x option" {
-        set cmd "src/redis-cli -h [srv host] -p [srv port]"
+    if {!$::tls} {
```

Comment:
you have the `tls:skip` tag instead of this `if`.
i.e.
```tcl
test {name} {
} {val} {tls:skip}
```

Reply:
oh right, updated it. now i think we are good to go

Reply:
now that i think of it, i suppose the only reason you skip tls is because you don't pass the required arguments to redis-cli.
is there any reason not to use the infra in `support/cli.tcl`?
i see other parts of this test are doing `set cmdline [rediscli [srv host] [srv port] $opts]`

i don't think it matters for this specific test (coverage for TLS), but later people could copy this pattern.

Reply:
last time i used it, i always had a problem, until now. i suppose i just forgot it this time

Reply:
so now it works? (with TLS and all?)
ready for merge?

Reply:
yes, i tested it in my centos7 machine, with or without TLS
althought i did not test it in full CI, but i suppose it is ok
there is no problem with `rediscli`